orcas/l1l2.go

```diff
@@ -615,7 +636,33 @@ func (l *L1L2Orca) Get(req common.GetRequest) error {
 
 					if err != nil {
 						metrics.IncCounter(MetricCmdGetSetErrorsL1)
-						return err
+
```

Comment:
Not returning an error is the right thing.

However, the Delete operation may not be necessary.  The Get operation never modified L2.  And presumably L1 get failed because it wasn't there.  Inconsistency would only happen if there was some error that would cause the Set (after Get) to fail but in reality succeeded, while another concurrent modify operation had already completed.  We don't handle the concurrent modify operation condition anyway, so this really only reacts to a more rare condition.

All this makes me wonder if Add operation should have been used instead of Set after the Get Miss.